### PR TITLE
Use 'dautotest' target for transparency and easy updates

### PR DESCRIPTION
**Pull request by wilzbach**
I wasn't sure whether I can remove the legacy checks.

See also: https://github.com/dlang/dlang.org/pull/2031

### CODE REVIEW

**Comment by CyberShadow** [comment]
> I wasn't sure whether I can remove the legacy checks.

No, we should keep them. Digger attempts to be able to build any reasonably recent version of D, and that extends to the website. See the existing checks (incl. the one removed here) for an example. Checking if the makefile contains "dautotest" should work fine as a way to detect the target's presence.

**Comment by wilzbach** [comment]
> See the existing checks (incl. the one removed here) for an example. Checking if the makefile contains "dautotest" should work fine as a way to detect the target's presence.

Ok. Done. Reading the file four times in a row seemed like a bit of a waste, so I added one line to "cache" it.

**Comment by wilzbach** [comment]
image

You should probably enable "PR auto-cancellation" at Travis...

File: sys/net/wininet.d [hunk 1]
```diff
@@ -137,6 +138,11 @@ protected:
 	}
 
 	final static void doDownload(HNet hUrl, void delegate(ubyte[]) sink)
+	{
+        doDownload(hUrl, cast(void delegate(const ubyte[])) sink);
```
Comment (CyberShadow) [inline sys/net/wininet.d]: ae uses tabs for indentation.

File: sys/d/manager.d [hunk 2]
```diff
@@ -1719,16 +1719,18 @@ EOS";
 					[
 						config.build.components.website.diffable
 						? ["all", "verbatim", "pdf"] ~ (
-							makeFullName.readText.indexOf("diffable-intermediaries") >= 0
+							makeFullText.indexOf("dautotest") >= 0
```
Comment (CyberShadow) [inline sys/d/manager.d]: This doesn't look like it's doing what you meant. `dautotest` already includes `all`, `verbatim` and `pdf`, doesn't it?

**Comment by CyberShadow** [comment]
Looks like fixing Appveyor is going to take more hammering. Otherwise I think this is ready.

File: sys/d/manager.d (hunk 2 above)
Comment (wilzbach) [reply sys/d/manager.d]: Oh sorry. I accidentally reverted that bit :/

File: sys/net/wininet.d (hunk 1 above)
Comment (wilzbach) [reply sys/net/wininet.d]: If you add an `.editorconfig`, it avoids problems with dumb people like myself.

**Pull request merged**